src/lexer.coffee

```diff
@@ -667,6 +667,9 @@ exports.Lexer = class Lexer
         @error message, origin[2] if message
       return value.length if skipToken
 
+    if prev?[0] is 'IMPORT' and value is '('
```

Comment:
```suggestion
    if value is '(' and prev?[0] is 'IMPORT'
```

Like the next `if` statement, I think this would be faster if we put the `value` check first (as it’ll return false a lot more often than `prev?` will).

Reply:
@GeoffreyBooth sure

"Committed suggestion" - this is the first time I've seen that Github feature, neat! The only thing that feels weird about it (in general) is committing without checking locally that tests are passing - but I guess with this workflow you just watch CI afterwards?

Reply:
Actually, realized (in pulling the updated branch) that since we commit `lib/`, the "Commit suggestion" workflow also isn't complete since I needed to recompile locally and push an updated `lib/`

But it's still a cool way of specify suggestions/changes 👍